test/modules/lydia/inFunc.chpl

```diff
@@ -6,5 +6,6 @@ module OuterModule {
     }
   }
 
+  use M;
```

Comment:
Yes the `use M` here is not required for the current behavior. I added it because I was looking at a lot of other tests where errors aren't emitted for dead modules, because IMO the only thing we need to test here is the behavior when the module is live.

I can remove this `use M` if you feel strongly about it.

Reply:
As long as the error is emitted either way, this doesn't bother me.